ts/Stock/Indicators/IKH/IKHIndicator.ts

```diff
@@ -0,0 +1,960 @@
+                     */
+                    lineColor: void 0
+                }
+            },
+            /**
+             * The styles for Senkou Span B line
+             */
+            senkouSpanB: {
+                styles: {
+                    /**
+                     * Pixel width of the line.
+                     */
+                    lineWidth: 1,
+                    /**
+                     * Color of the line.
+                     *
+                     * @type {Highcharts.ColorString}
+                     */
+                    lineColor: void 0
+                }
+            },
+            /**
+             * The styles for area between Senkou Span A and B.
+             */
+            senkouSpan: {
+                /**
+                 * Color of the area between Senkou Span A and B,
+                 * when Senkou Span A is above Senkou Span B. Note that if
+                 * a `style.fill` is defined, the `color` takes precedence and
+                 * the `style.fill` is ignored.
+                 *
+                 * @see [senkouSpan.styles.fill](#series.ikh.senkouSpan.styles.fill)
+                 *
+                 * @sample stock/indicators/ichimoku-kinko-hyo
+                 *         Ichimoku Kinko Hyo color
+                 *
+                 * @type      {Highcharts.ColorString|Highcharts.GradientColorObject|Highcharts.PatternObject}
+                 * @since     7.0.0
+                 * @apioption plotOptions.ikh.senkouSpan.color
+                 */
+
+                /**
+                 * Color of the area between Senkou Span A and B,
+                 * when Senkou Span A is under Senkou Span B.
+                 *
+                 * @sample stock/indicators/ikh-negative-color
+                 *         Ichimoku Kinko Hyo negativeColor
+                 *
+                 * @type      {Highcharts.ColorString|Highcharts.GradientColorObject|Highcharts.PatternObject}
+                 * @since     7.0.0
+                 * @apioption plotOptions.ikh.senkouSpan.negativeColor
+                 */
+
+                styles: {
+                    /**
+                     * Color of the area between Senkou Span A and B.
+                     *
+                     * @deprecated
+                     * @type {Highcharts.ColorString|Highcharts.GradientColorObject|Highcharts.PatternObject}
+                     */
+                    fill: 'rgba(255, 0, 0, 0.5)'
+                }
+            },
+            dataGrouping: {
+                approximation: 'ichimoku-averages'
+            }
+        } as IKHOptions);
+    /* *
+    *
+    *  Properties
+    *
+    * */
+
+    public data: Array<IKHPoint> = void 0 as any;
+    public options: IKHOptions = void 0 as any;
+    public points: Array<IKHPoint> = void 0 as any;
+    public pointValKey = 'tenkanSen';
+    public nameComponents = ['periodSenkouSpanB', 'period', 'periodTenkan'];
+
```

Comment:
The last two properties should be kept in the extend call for the class prototype.

Reply:
Moved those two properties to extend call